app/celery/letters_pdf_tasks.py

```diff
@@ -71,22 +71,22 @@ def create_letters_pdf(self, notification_id):
             dao_update_notification(notification)
 
         current_app.logger.info(
-            'Letter notification reference {reference}: billable units set to {billable_units}'.format(
-                reference=str(notification.reference), billable_units=billable_units))
+            'Letter notification reference %s: billable units set to %s', str(notification.reference), billable_units
```

Comment:
Is billable_units a string of a number?  If it's the latter, you should use `%d` (integer type) or otherwise.  ([ref](https://docs.python.org/3.8/library/string.html?highlight=format%20string#format-specification-mini-language))

Reply:
changed to %d